shoebot/grammar/grammar.py

```diff
@@ -305,6 +305,7 @@ def run(
             else:
                 errmsg = simple_traceback(e, executor.known_good or "")
             print(errmsg, file=sys.stderr)
+            raise ShoebotScriptError()
```

Comment:
Do we need the next lines?  It looks like they will never run.

Reply:
good find, turns out break_on_error was never used, so just took it out.